pubspec.yaml

```diff
@@ -12,20 +12,20 @@ dependencies:
     sdk: flutter
   package_info: ^0.4.0+18
   chewie: ^0.9.10
-  video_player: ^0.10.11
-  flutter_widget_from_html: ^0.3.3+3
+  video_player: ^0.10.11+1
+  flutter_widget_from_html: ^0.3.3+4
   matrix_gesture_detector: ^0.1.0
-  url_launcher: ^5.4.7
+  url_launcher: ^5.4.10
   flutter_svg: ^0.17.4
-  shared_preferences: ^0.5.7+2
+  shared_preferences: ^0.5.7+3
   intl: ^0.16.1
   photo_view: ^0.9.2
   provider: ^4.1.2
-  firebase_core: ^0.4.4+3
-  firebase_analytics: ^5.0.11
+  firebase_core: ^0.4.5
```

Comment:
If the major version number is 0, then it means the version is not stable yet. ^0.4.4 is equivalent to '>=0.4.4 <0.5.0'.

Reply:
yep see above .